meta-lmp-bsp/recipes-core/base-files/base-files/imx8mqevk/fstab

```diff
@@ -0,0 +1,8 @@
+/dev/root            /                    auto       defaults              1  1
+proc                 /proc                proc       defaults              0  0
+devpts               /dev/pts             devpts     mode=0620,gid=5       0  0
+tmpfs                /run                 tmpfs      mode=0755,nodev,nosuid,strictatime 0  0
+tmpfs                /var/volatile        tmpfs      defaults              0  0
+
+# boot partition
+/dev/mmcblk0p1       /mnt/boot            vfat       noatime,sync          0  0
```

Comment:
Here you have the hardcoded path for /boot, but at lmp-machine-custom.inc it seems the rootfs is not hardcoded (probably using the label by default), so there is a mismatch.

If we know the mmc device node id, probably better to also set root= to that as well.

Reply:
changed root= param in lmp-machine-custom.inc